### PR TITLE
Move server sku to a variable

### PR DESCRIPTION
**Pull request by neb42**

### CODE REVIEW

File: deploy/terraform/variables.tf [hunk 1]
```diff
@@ -283,4 +283,12 @@ EOF
     condition     = contains(["debug", "info", "warn", "error"], var.log_level)
     error_message = "The log level must be one of \"debug\", \"info\", \"warn\", or \"error\"."
   }
+}
+
+variable "service_plan_sku_name" {
+  type        = string
+  default     = "P1v2"
+  description = <<EOF
+THe sku name of the app service plan.
```
Comment (internetti) [inline deploy/terraform/variables.tf:292]: would it make sense to have a list of valid values here?

Comment (ludydoo) [reply deploy/terraform/variables.tf:292]: That would be a lot of information and is subject to change. You can see the skus here: https://azure.microsoft.com/en-us/pricing/details/app-service/windows/

Comment (ludydoo) [reply deploy/terraform/variables.tf:292]: Just as a suggestion, make the terraform variables more explicit. In this case `app_service_plan_sku_name`

Comment (neb42) [reply deploy/terraform/variables.tf:292]: I think the name is good as is because the terraform resource is called service_plan

**Pull request merged**